UKHomeOffice/nmsw-ui
# NMSW-213 Add back button

# Ticket

NMSW-213

----

## AC

The link should be placed at the top of a page, before the main element
The link must be contained within a landmark for accessibility
The link should take users to the previous page they were on, in the state they last saw it.  

----

## To test

- Run the app
- Go to localhost:3000
- Go through the app so far
- There **should not** be back links on the:
- > Landing page ( / )
- > Sign in page ( /sign-in )
- > Check your email page - registration process ( /create-account/check-your-email )
- > Your email address has been verified page - registration process ( /create-account/email-verified)
- > Any confirmation page with the bigger green panel e.g. after creating an account or submitting the form on /second-page (/confirmation **or** /create-account/account-created )
- > Dashboard (/dashboard )
- > Footer links ( /cookies **or**  /privacy-notice **or** /accessibility-statement
- There **should** be links on all other pages such as forms

----

## Notes

- Just handles showing the user a back link on form pages and not showing it on other pages. In RR6 the back link works the same as the browser back button
- This does not have any formData management as that was handled in NMSW-119 
- > (However, will need to add it once the API is in place as we will want to persist data on refresh but clear/reset the data on the back click - need a separate ticket)
